radar/radar.go

```diff
@@ -77,7 +90,10 @@ func (radar *Radar) Scanner(logger lager.Logger, resourceName string) ifrit.Runn
 			case <-signals:
 				return nil
 
-			case <-ticker.C:
+			case newInterval := <-radar.updateIntervalChan:
```

Comment:
channel seems a bit weird for this since it's always a synchronous function call that's populating it (which is why it has to be buffered, otherwise it'd deadlock)